src/main/java/com/zendesk/maxwell/row/RowEncrypt.java

```diff
@@ -34,13 +34,13 @@ public static String encrypt(String value, String key, String initVector) {
 
 	public static String decrypt(String value, String key, String initVector) {
 		try {
-			IvParameterSpec ivSpec = new IvParameterSpec(initVector.getBytes("UTF-8"));
-			SecretKeySpec skeySpec = new SecretKeySpec(key.getBytes("UTF-8"), "AES");
+			IvParameterSpec ivSpec = new IvParameterSpec(initVector.getBytes("ASCII"));
+			SecretKeySpec skeySpec = new SecretKeySpec(key.getBytes("ASCII"), "AES");
 
 			Cipher cipher = Cipher.getInstance("AES/CBC/PKCS5PADDING");
 			cipher.init(Cipher.DECRYPT_MODE, skeySpec, ivSpec);
 
-			return new String(cipher.doFinal(Base64.decodeBase64(value.getBytes("UTF-8"))), Charset.forName("UTF-8"));
+			return new String(cipher.doFinal(Base64.decodeBase64(value.getBytes("ASCII"))), Charset.forName("ASCII"));
```

Comment:
The _second_ one on this line might need to stay UTF-8, I think? i.e. the base64 string is ASCII, but the bytes it represents are UTF-8.

To match, I think you should also change the `value.getBytes()` in `encrypt` (line 25) to use UTF-8, since you need to make sure the roundtrip to base64 and back is using the same byte encoding.